.changelog/unreleased/bug-fixes/1504-timeout_check.md

```diff
@@ -1,2 +0,0 @@
-- Fix for packet timeout computation
```

Comment:
If I recall correctly, what we usually do is move these files out of `.changelog/unreleased` directory to the appropriate version directory (in this case, `.changelog/v0.8.0/`. Should we consider doing the same for this file?

Reply:
Thanks for catching that! 👍 I forgot that step since I did this manually. Fixed in https://github.com/informalsystems/ibc-rs/pull/1528/commits/1ad3068709900d69d2f43b1890f460424d3831df.